samples/msal-node-samples/standalone-samples/auth-code/readme.md

```diff
@@ -0,0 +1,203 @@
+
+Once MSAL Node is installed, and you have the right files, come here to learn about this scenario.
+
+### How is this scenario used?
+The Auth Code flow is most commonly used for a web app that signs in users.  General information about this scenario is available [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-web-app-sign-user-overview?tabs=aspnetcore).
+
+## Test the Sample
+
+### Configure the application
+Open the `index.js` file.
+
+Find the `config` object.  We will change this to add details about our app registration and deployment.
+
+By default, this configuration is set to support all Microsoft accounts. This includes Azure AD accounts used by organizations, and MSA accounts typically used by consumers. 
+
+Before proceeding, go to the Azure portal, and open the app registration for this app.
+
+#### **Client ID**
+Within the "Overview" you will see a GUID labeled **Application (client) ID**.  Copy this GUID to the clientId field in the config.
+
+Click the **Authentication** link in the left nav.
+
+#### **Authority**
+Check that supported account types are: **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+
+If so, then leave the default setting for authority as `https://login.microsoftonline.com/common`
+
+For other supported account types, review the other [Authority options](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration).  Unless there is a specific need to restrict users of your app to an organization, we strongly suggest that everyone use the default authority.  User restrictions can be placed later in the application flow if needed.
+
+#### **Client Secret**
+
+This secret helps prevent third parties from using your app registration.
+Click on `Certificates and Secrets` in the left nav.
+Click `New Client Secret` and pick an expiry.
+Click the `Copy to Clipboard` icon, and add the secret to the config object in index.js.
+
+🎉You have finished the basic configuration!🎉
+
+### Executing the application
+
+From the command line, let npm install any needed dependencies.  This only needs to be done once.
+
+```bash
+$ npm install
+```
+1. Once the dependencies are installed, you can run the sample application by using the following command:
+
+```bash
+$ npm start
+```
+
+2. Navigate to http://localhost:3000 (or whatever port number specified) with the browser of your choice.
+
+### Customizing the application
+
+To customize the start script, review the `package.json` file.
+
+## Adding this scenario to an existing application
+
+### Import the Configuration Object
+
+If you set up the sample with your app registration, you may be able to copy this object directly into your application.  
+
+
+```js
+const config = {
+    auth: {
+        clientId: "12d77c73-d09d-406a-ae0d-3d4e576f7d9b",
```

Comment:
Do we want to keep this `GUID`? Or have the user fill it?

Reply:
the guidance does indicate to replace the sample ID.  Since this is a cca, they would not have the secret anyways, so I have no concerns keeping it

Reply:
I prefer the "sample data" approach to the "ENTER ID HERE".  It maintains more context.

Reply:
The only advantage with a `placeholder string` is that it fails with `invalid client_id` forcing the user to add it.

Reply:
so we can add the string _placeholder after the GUID?